src/utils/worker/sheep.ts

```diff
@@ -0,0 +1,72 @@
+import { prisma } from '../prisma';
+import { GoalCommentCreateSchema } from '../../schema/goal';
+
+const sheepEmail = process.env.SHEEP_EMAIL || 'sheep@taskany.ru';
+const sheepPassword = process.env.SHEEP_PASSWORD || 'taskany';
+const sheepAvatar = process.env.SHEEP_AVATAR;
```

Comment:
We can register sheep on Gravatar ;) And use /public/sheep.png as default avatar.